### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bunilly**

### CODE REVIEW

**Comment by benjaminfan** [comment]
We probably want more information and setup steps for fastapi

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # data-structures
-Example Data-Structures for Students
+Datastructure and complex algorithms behavior backend visualization service in graphql
```
Comment (bunilly) [inline README.md]:
```suggestion
Data-structure traversal and algorithm behavior backend visualization service in graphql

## Steps to Run
todo: fill out
```

Comment (bunilly) [reply README.md]: Is this what you were thinking?

Comment (benjaminfan) [reply README.md]:
```suggestion
Data-structure traversal and algorithm behavior backend visualization service in graphql

## Steps to Run
### Docker
1. Build - `make build`
1. Run - `make run` 

### Local
1. `uvicorn src.server.app:app --reload --workers 1 --host 0.0.0.0 --port 5050`
```

**Review by benjaminfan — APPROVED** [review]

**Pull request merged**